libsmtutil/SMTLib2Interface.cpp

```diff
@@ -38,11 +38,11 @@ using namespace solidity::frontend;
 using namespace solidity::smtutil;
 
 SMTLib2Interface::SMTLib2Interface(
-	map<h256, string> const& _queryResponses,
```

Comment:
Why is this changing to non-const?

Reply:
Yep

Reply:
I would actually suggest to merge this API refactoring separately, it could be merged right away as it is just refactoring the API.

Reply:
But does it have to? I mean, if it's already reviewed as part of this one... :p
The thing is. Those API changes are needed for whatever code that wants to use SMTPortfolio without coming from the CompilerStack which would forward the query responses and callback. So in a way it is related

Reply:
Well, then make it a distinct commit?

Reply:
I'll extract this.